recipes/fgt/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+mkdir build
+cd build
+cmake -G "Unix Makefiles" \
+      -DCMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
+      -DCMAKE_BUILD_TYPE:STRING=Release \
+      -DENABLE_TESTS=OFF \
+      -DCMAKE_LIBRARY_PATH="${PREFIX}/lib" \
+      -DCMAKE_INCLUDE_PATH="${PREFIX}/include" \
+      ..
+
+# CircleCI offers two cores.
+make -j $CPU_COUNT
+make install
```

Comment:
Is there a make check/test we can run here?

Reply:
There is. I can add to the feedstock.